build/templates/_converters.py.mako

```diff
@@ -222,6 +222,36 @@ def convert_init_with_options_dictionary(values, encoding):
     return init_with_options_string
 
 
+% if config['module_name'] == 'nitclk':
+# nitclk specific converters
+def convert_to_nitclk_session_num(item):
```

Comment:
The term "NI-TClk Session Num" doesn't exist anywhere, right?

In our driver APIs in which NI-TClk is supported we have a VI called "Get Session Reference".
In C you just pass the ViSession.

I think `convert_session_reference_to_vi_session` would be pretty self explanatory.